apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -0,0 +1,207 @@
+import { html, HtmlSafeString } from '@prairielearn/html';
+import { nodeModulesAssetPath, compiledScriptTag } from '../lib/assets';
+import { EncodedData } from '@prairielearn/browser-utils';
+
+export function QuestionsTable(
+  questions,
+  showAddQuestionButton,
+  current_course_instance,
+  course_instances,
+  urlPrefix,
+  plainUrlPrefix,
+  errorMessage,
+): HtmlSafeString {
```

Comment:
By convention, components should take a single object with as many named properties as needed. Don't forget to add types to these parameters!

Reply:
done, thanks